### DISCUSSION TITLE
Logo questions

### DISCUSSION THREAD

**Discussion by aforsythe** [squidfunk/mkdocs-material#3534] [Q&A] [answered]
Just a few quick logo questions ... 

1. Is there a way to simply change the `href` url when the logo is clicked?  An override?
2. When changing the Logo to a `.png` I'm seeing the logo flash when preforming an action that reloads the page, like the image is loading slowly.  Is there a way to avoid this?

thanks

**Comment by squidfunk** [comment, marked as answer]
Regarding 1.: Yes, as stated in the documentation:
https://squidfunk.github.io/mkdocs-material/setup/changing-the-logo-and-icons/#logo

> Normally, the logo in the header and sidebar links to the homepage of the documentation, which is the same as site_url. This behavior can be changed with the following configuration:
>
> ```
> extra:
>  homepage: https://example.com
> ```

Regarding 2.: Yes. Set appropriate cache headers on your web server, so the browser cache it. Note that the built-in dev server does not set those headers, which is why the logo might flash on `mkdocs serve`.

**Reply by aforsythe** [reply]
Thank you so much  ... missed the `homepage` configuration option.  The excellent documentation is deceivingly simple but so full of great information, I read right over it.
